docker/intel-dgpu-driver.Dockerfile

```diff
@@ -27,7 +27,8 @@ LABEL name='intel-data-center-gpu-driver-container'
 LABEL summary='Intel® Data Center GPU Driver Container Image'
 LABEL description='Intel® Data Center GPU Driver container image for Red Hat OpenShift Container Platform'
 
```

Comment:
Could you please explain more about "so we are doing the microdnf update so that we get the latest
updates not in the base image."?

Reply:
microdnf update updates the packages in the base image to its latest release. For example the base image has package named x and release 1.0. but after the base image is released the package x has an update release of 1.1. so this updates the package to 1.1.

Reply:
I updated the commit message as discussed.